serverFiles/app.js

```diff
@@ -19,10 +22,20 @@ app.get('/api/articles/:article_id', getArticlesById);
 
 app.get('/api/articles/:article_id/comments', getCommentsByArticleId);
 
+app.post('/api/articles/:article_id/comments', postCommentByArticleId);
+
 app.all('*', (req, res) => {
   res.status(404).send({ msg: 'Not found' });
 });
 
+app.use((err, req, res, next) => {
+  if (err.code) {
+    res.status(404).send({ msg: 'Not found' });
```

Comment:
While this works here it might be nice to move this error handler out to a seperate file to keep the app clear.